diff --git a/ios/RNFBConfig/RNFBConfigModule.mm b/ios/RNFBConfig/RNFBConfigModule.mm
index 230435f3422775649f4edef87dd2a7876be4a0b8..68b2f593b7b4a3d583d8020c007db773638a6994 100644
--- a/ios/RNFBConfig/RNFBConfigModule.mm
+++ b/ios/RNFBConfig/RNFBConfigModule.mm
@@ -38,13 +38,13 @@ - (void)invalidate {
   return std::make_shared<facebook::react::NativeRNFBTurboConfigSpecJSI>(params);
 }
 
-- (facebook::react::ModuleConstants<JS::NativeRNFBTurboConfig::Constants::Builder>)
+- (facebook::react::ModuleConstants<JS::NativeRNFBTurboConfig::Constants>)
     constantsToExport {
   return [_RCTTypedModuleConstants
       newWithUnsafeDictionary:[RNFBConfigHelper getConstantsForAppName:DEFAULT_APP_DISPLAY_NAME]];
 }
 
-- (facebook::react::ModuleConstants<JS::NativeRNFBTurboConfig::Constants::Builder>)getConstants {
+- (facebook::react::ModuleConstants<JS::NativeRNFBTurboConfig::Constants>)getConstants {
   return [self constantsToExport];
 }
 
